modules/virt-single-node-cluster.adoc

```diff
@@ -9,4 +9,3 @@ You can use {VirtProductName} with single node clusters, also known as Single No
 
 * {VirtProductName} components have only one replica, and pod disruption budgets do not apply.
 * Live migration is not supported.
-* Due to differences in storage behavior, some virtual machine templates are incompatible with SNO. To ensure compatibility, do not set the `evictionStrategy` field for any templates or virtual machines that use data volumes or storage profiles.
```

Comment:
I recently saw this posted in team-docs: 

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#single-node-openshift

"single-node OpenShift
Usage: single-node OpenShift

Single-node OpenShift (or {product-title} on a single-node cluster) is a deployment footprint that provides control plane and worker node capabilities in a single server for deployments in constrained environments.

Do not use: Single Node Openshift (SNO)."

Changing this everywhere is probably out of scope for this PR, but if you could change it here (and in the intro sentence), that would be great.